### DISCUSSION TITLE
Get started

### DISCUSSION THREAD

**Discussion by WNTBq** [reactivemarkets/react-financial-charts#608] [Q&A]
Hello,
i wanted to add the BasicAreaSeries.ts  to another project. I was working with "react-stockcharts" before and i have a working project where i would like to replace the "AreaChart" with the "BasicAreaSeries" since "react-stockcharts" is more or less "abandoned" and only working with react <=16. So i was very happy to find this project. 
However i have problems getting started. I got the "react-financial-charts"-project running (build from Source), but i can not isolate the BasicAreaSeries to my project. 
My naive approach was to copy the code of "BasicAreaSeries.tsx", install the dependencies, update the paths for the data, that i copied too.
Allready in the editor i get the error "generic Typ "ChartCanvas<TXAxis>" requires 1 Typeargument".
So it seems a argument is missing.  However reviewing the source code i couldnt find any mandatory argument that i miss.
```xml
<ChartCanvas      
                height={height}
                ratio={ratio}
                width={width}
                margin={this.margin}
                data={data}
                displayXAccessor={displayXAccessor}
                seriesName="Data"
                xScale={xScale}
                xAccessor={xAccessor}
                xExtents={xExtents}

                <Chart id={1} yExtents={this.yExtents}>
                    <AreaSeries yAccessor={this.yAccessor} {...rest} />
                    <XAxis />
                    <YAxis />
                </Chart>
            </ChartCanvas>
```

I have only rudimentary knowlege about Typescript and  Storybook.
Some examples with a minimum of features would be great to get started..

Any help would be appreciated.

Thanks and best regards!

**Comment by markmcdowell** [comment]
You could play around with this https://codesandbox.io/s/react-financial-charts-demo-c88wz

**Comment by WNTBq** [comment]
Thanks. I saw this example but i noticed that it is still react 16 and was not sure it will work for me, - i will try ..
